code/dqc/tablecomp.q

```diff
@@ -0,0 +1,5 @@
+/- only meant to be used for comparison
+\d .dqc
+tablecomp:{[tab]
+  (1b;("table count of ",(string tab)," is ",count tab);count get tab)
```

Comment:
If `tab` is a symbol then `count tab` will be 1, should this be:
```suggestion
  (1b;("table count of ",(string tab)," is ",c);c:count get tab)
```
And I'm guessing this should always be true as if it can't read the table it will fail?

Reply:
Yes, if it cannot read the table it will fail through dqc's error handling